cmd/manager/main.go

```diff
@@ -144,6 +157,7 @@ func main() {
 	registerComponentOrExit(mgr, argoapi.AddToScheme)
 	registerComponentOrExit(mgr, configv1.AddToScheme)
 	registerComponentOrExit(mgr, monitoringv1.AddToScheme)
+	registerComponentOrExit(mgr, oauthv1.AddToScheme)
```

Comment:
Move this inside the if-clause `util.IsKeycloakAPIAvailable()`?

Reply:
I think we don't need any condition to register the scheme, I have removed the if-clause and registered both keycloak and Oauth schemes.